Examples/Physics_applications/spacecraft_charging/PICMI_inputs_rz.py

```diff
@@ -275,6 +275,13 @@ def compute_actual_charge_on_spacecraft():
     warpx_file_prefix="spacecraft_charging_plt",
 )
 
+part_scraping_boundary_diag = picmi.ParticleBoundaryScrapingDiagnostic(
```

Comment:
Comment by @roelof-groenewald:
Careful, the sim here uses the boundary buffer to accumulate but the diag cleans out the buffer on dump.

Could either
- use another example or
- only do the dump at the end or
- add an option to not clean out the buffer on dump (should _not_ be default-on)

Reply:
@RemiLehe suggests adding this in the magnetostatic w/ boundary example